### PR TITLE
feature: adding translation to brazilian portuguese

### PR DESCRIPTION
**Pull request by nikolai1312**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [X] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Adding translation to brazilian portuguese for the app

## Featured Covered in this PR

- [X] Task 1
- [ ] Task 2

## Related Tickets & Documents

- Related Issue #75 
- Closes #

## Tested Feature??

- [ ] In Real Device.
- [X] In Emulator

### CODE REVIEW

**Review by micedreams — DISMISSED** [review]
The changes looks good to me!! Good job!!

File: lib/app/view/app.dart [hunk 1]
```diff
@@ -113,6 +113,7 @@ class _AppViewState extends State<AppView> {
             Locale('pa'),
             Locale('he'),
             Locale('kn'),
+            Locale('pt', 'BR'),
```
Comment (SankethBK) [inline lib/app/view/app.dart:116]: I think this should be `pt-br`

**Comment by nikolai1312** [comment]
Fixed right now, thanks for replying.

File: lib/app/view/app.dart [hunk 2]
```diff
@@ -113,6 +113,7 @@ class _AppViewState extends State<AppView> {
             Locale('pa'),
             Locale('he'),
             Locale('kn'),
+            Locale('pt', 'br'),
```
Comment (SankethBK) [inline lib/app/view/app.dart:116]: Sorry for previous comment, didn't know Flutter ignores country code for filename. It works if we change this to
```
            Locale('pt', "BR"),
```

File: lib/l10n/arb/app_pt-br.arb [hunk 3]
```diff
@@ -0,0 +1,82 @@
+{
+  "language":"Português - Brasil",
```
Comment (SankethBK) [inline lib/l10n/arb/app_pt-br.arb:2]: Change this file's name to `lapp_pt.arb`, (Sorry again for my previous comment, this one will work)

**Comment by nikolai1312** [comment]
Fixed

**Review by SankethBK — APPROVED** [review]
LGTM, thanks for contributing!

**Pull request merged**